### ISSUE TITLE
Add pictures to OSM objects

### ISSUE THREAD

**Issue by pyrog** [organicmaps/organicmaps#8398]
**Is your feature request related to a problem?**
No. I just want to take pictures of an existing OSM object or a place and share them to an online platform.

**Describe the ideal solution**
- For an existing OSM object, select it first on the map.
- For the current location, do… nothing.
- Take one or more photos from the Organic Map or select pictures in the gallery.
- Write OSM object id in the picture's metadata (i.e. `https://osm.org/node/1234`)
- Upload picture(s) to the platform.
- Get id of each picture and add/edit the OSM object "picture" tag according to the platform.
i.e use the `wikimedia_commons` tag if the platform is **Wikimedia Commons**, `panoramax` for **Panoramax**, and so on.

**Describe alternatives you have considered**
- Organic Maps could only take pictures, optionally upload them.<BR>The user could use any editor later to "link" the OSM object to the picture(s).

- Organic Maps could let the user to choose one platform in the settings, use only one platform, or none.<BR> In the last case, pictures are stored in the smartphone's (internal or external) memory.

- Organic Maps could handle directly several platforms or use a "proxy" the handle the code for each platform.

Related to #5469

**Comment by pktiuk** [comment]
I don't think this is a good idea for current vesion of Organic maps. Firstly it should be implementing showing images (like the ones from wikipedia https://github.com/organicmaps/organicmaps/issues/8922) and then we could think about uploading anything.
